src/SpacetimeDBClient.cs

```diff
@@ -126,7 +160,7 @@ struct ProcessedMessage
             public ServerMessage message;
             public List<DbOp> dbOps;
             public DateTime timestamp;
-            public ReducerEvent? reducerEvent;
+            public EventContext reducerEvent;
```

Comment:
Does this one really not need to be nullable anymore? I think there was some case in which a reducer we returned was `null` (if it's unknown type maybe?).

Reply:
Good point, Ill check